workflow/rules/candidate_calling.smk

```diff
@@ -2,6 +2,7 @@ rule freebayes:
     input:
         ref="resources/genome.fasta",
         ref_idx="resources/genome.fasta.fai",
+        regions="results/regions/{group}.bed",
```

Comment:
For WGS, the region file is superfluous. Hence, we should make it switchable in the config. If it is switched of, a function here can return `[]`.